src/matter/cluster/LabelCluster.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ArrayT, Field, ObjectT, StringT } from "../../codec/TlvObjectCodec";
+import { AccessLevel, Attribute, Cluster, WritableAttribute } from "./Cluster";
+import { MatterCoreSpecificationV1_0 } from "../../Specifications";
+
+/**
+ *
+ * This is a string tuple with strings that are user defined.
+ *
+ * @see {@link MatterCoreSpecificationV1_0} § 9.7.5.1
+ */
+const LabelT = ObjectT({
+    /** Contains a string as label without a further defined semantic n this base cluster. */
+    label: Field(0, StringT( { length: 16 } )), /* default: "" */
+    /** Contains a string as value without a further defined semantic n this base cluster. */
+    value: Field(1, StringT( { length: 16 } )), /* default: "" */
+});
+
+/**
+ * This cluster provides a feature to tag an endpoint with zero or more labels.
+ * Derived from LabelCluster ({@link MatterCoreSpecificationV1_0} § 9.7)
+ *
+ * clusterRevision: 1
+ *
+ * @see {@link MatterCoreSpecificationV1_0} § 9.9
+ */
+export const UserLabelCluster = Cluster({
+    /** Is a base cluster, so no id */
```

Comment:
This comment is incorrect now... delete?

Reply:
copy/paste error ;-(